source/shared/cpp/ObjectModel/BaseElement.h

```diff
@@ -98,6 +99,8 @@ namespace AdaptiveSharedNamespace
     protected:
         virtual void PopulateKnownPropertiesSet();
         void SetTypeString(const std::string& type) { m_typeString = type; }
+        void SetCanFallbackToAncestor(bool value) { m_canFallbackToAncestor = value; }
```

Comment:
Doesnt this also need to include the requires property on all base elements ? Are we doing that as well?

Reply:
sure, i will ensure that when it's merged with master, and implements requies

---
In reply to: [275519519](https://github.com/Microsoft/AdaptiveCards/pull/2675#discussion_r275519519) [](ancestors = 275519519)